Sources/NIOSSL/SSLContext.swift

```diff
@@ -569,6 +567,40 @@ extension NIOSSLContext {
             parentSwiftContext.keyLogManager!.log(linePointer)
         }
     }
+    
+    /// Takes a path and determines if the file at this path is of c_rehash format .
+    internal static func _isRehashFormat(path: String) throws -> Bool {
+        // Check if the element’s name matches the c_rehash symlink name format.
+        // Note that `lastPathComponent` is not available here.
+        guard let lastPathComponent = path.split(separator: "/").last else { return false }
+        let filenameElements = lastPathComponent.split(separator: ".")
+        
+        guard filenameElements.count == 2,
+              let filenameElement = filenameElements.first,
+              let fileExtensionElement = filenameElements.last else {
+            return false
+        }
+        
+        // The links created are of the form HHHHHHHH.D, where each H is a hexadecimal character and D is a single decimal digit.
+        let filename = String(filenameElement)
+        let fileExtension = Int(fileExtensionElement)
+        
+        // Double check that the extension did not fail to cast to an integer.
+        // Make sure that the filename is an 8 character hex based file name.
+        guard fileExtension != nil,
```

Comment:
I believe the file extension is required to be `0`, so we could just do a String comparison instead of turning it into an Int.